src/components/Header/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import './index.scss';
+
+import logo from '@src/assets/images/logo.svg';
+import Lv1 from '@src/assets/images/Lv1.svg';
+import headerItems from './headerItems';
+
+export interface IHeaderProps {
+  isLogin: boolean;
+}
+
+export interface IHeaderItems {
+  name: string;
+  route: string;
+}
+
+const Header: React.FC<IHeaderProps> = ({ isLogin = false }) => {
+  return (
+    <header className="headerContainer">
+      <div className="logoWrapper">
+        <img className="logoImage" src={logo} alt="logoImage" />
+      </div>
+      <div className="buttonContainer">
+        <ul className="linkButtonList">
+          {headerItems.map(({ name, route }: IHeaderItems) => {
+            return (
+              <li className="linkButton" key={name}>
+                {name}
+              </li>
+            );
+          })}
```

Comment:
```suggestion
          {headerItems.map(({ name, route }: IHeaderItems) => (
              <li className="linkButton" key={name}>
                {name}
              </li>
          ))}
```
이렇게 작성하는 건 어떨까요? 처음에 `return`이 2개여서 0.1초 정도 뭐지? 싶었어요 😢

Reply:
이 부분은 바로 수정하겠습니다! 👍